source/admin_manual/list_of_events.rst

```diff
@@ -2724,6 +2727,28 @@ emit an event.
 | deleted_count         | The count of deleted keys for the empty bucket.      |
 +-----------------------+------------------------------------------------------+
 
+.. _fs_object_write_uncertain:
+
+fs_object_write_uncertain
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: v2.4.0;v3.0.0
+
+The event is sent whenever an object write is uncertain.
+When HTTP write operation times out actual outcome is uncertain.
```

Comment:
@Sergey-Kitov could you change one more thing: "When HTTP write operation" -> "When a write HTTP operation" to make it clearer it's not a "HTTP WRITE" operation. Plus missing article.